pkgs/tools/misc/yubico-piv-tool/default.nix

```
@@ -41,32 +41,37 @@ stdenv.mkDerivation (finalAttrs: {
 
   buildInputs = [
     openssl
-    check
     zlib.dev
   ]
   ++ (if withApplePCSC then [ PCSC ] else [ pcsclite ]);
 
   cmakeFlags = [
-    "-DGENERATE_MAN_PAGES=ON"
-    "-DCMAKE_INSTALL_BINDIR=bin"
-    "-DCMAKE_INSTALL_INCLUDEDIR=include"
-    "-DCMAKE_INSTALL_MANDIR=share/man"
-    "-DCMAKE_INSTALL_LIBDIR=lib"
+    (lib.cmakeBool "GENERATE_MAN_PAGES" true)
+    (lib.cmakeFeature "BACKEND" (if withApplePCSC then "macscard" else "pcsc"))
+    (lib.cmakeFeature "CMAKE_INSTALL_BINDIR" "${placeholder "out"}/bin")
+    (lib.cmakeFeature "CMAKE_INSTALL_INCLUDEDIR" "${placeholder "out"}/include")
+    (lib.cmakeFeature "CMAKE_INSTALL_LIBDIR" "${placeholder "out"}/lib")
+    (lib.cmakeFeature "CMAKE_INSTALL_MANDIR" "${placeholder "out"}/man")
```

```suggestion
    (lib.cmakeFeature "CMAKE_INSTALL_MANDIR" "${placeholder "out"}/share/man")
```

Also wht is `placeholder "out"` added here?

---

@Aleksanaa Thanks for the review!
Oops I didn't realise I'd broken the manpage path.